### ISSUE TITLE
Compatibility with Angular 8

### ISSUE THREAD

**Issue by Jch71** [VadimDez/ng2-pdf-viewer#482]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

Hello, 

When I try to use PdfViewer with **Angular 8, compiling with Ivy**, I get :

Error: Type PdfViewerModule does not have 'ngModuleDef' property.
    at getNgModuleDef (VM8703 vendor.js:68747)
    at verifySemanticsOfNgModuleDef (VM8703 vendor.js:88543)
    at Array.forEach (<anonymous>)
    at verifySemanticsOfNgModuleDef (VM8703 vendor.js:88547)
    at Function.get (VM8703 vendor.js:88519)
    at getInjectorDef (VM8703 vendor.js:65866)
    at R3Injector.push../node_modules/@angular/core/__ivy_ngcc__/fesm5/core.js.R3Injector.processInjectorType (VM8703 vendor.js:80728)
    at VM8703 vendor.js:80615
    at VM8703 vendor.js:80928
    at Array.forEach (<anonymous>)

I guess the package is not ready yet ? 

When you build without Ivy, you don't get the error

**Comment by VadimDez** [comment]
Which `ng2-pdf-viewer` version did you use?

**Comment by VadimDez** [comment]
Fixed in `5.3.1`
